dallinger/recruiters.py

```diff
@@ -727,6 +736,60 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, participant: Participant):
+        """Screen-out a participant.
+
+        Parameters:
+            participant: Participant
+                The participant to screen-out.
+        """
+        if self.screen_out_allowed(participant):
+            try:
+                return self.prolificservice.screen_out(
+                    study_id=self.current_study_id,
+                    submission_id=participant.assignment_id,
+                    bonus_per_submission=participant.base_pay + participant.bonus,
+                    increase_places=self.config.get("auto_recruit"),
+                )
+            except ProlificServiceException as ex:
+                logger.exception(str(ex))
+
+    def screen_out_allowed(self, participant):
+        """Check if a participant can be screened out.
+
+        Returns False if:
+        - participant is not in 'working' status
+        - custom screening is enabled
+        - reward is below minimum required
+        """
+        if participant.status != "working":
+            return False
+
+        study = self.prolificservice.get_study(self.current_study_id)
+
+        if study.get("is_custom_screening", False):
+            raise ProlificRecruiterException(
+                f"Prolific study (ID {self.current_study_id}) doesn't allow screening-out of participants"
+            )
+
+        # Minimum wage thresholds: https://researcher-help.prolific.com/en/article/2273bd
+        prolific_min_wage_per_hour = 6
+        participants = Participant.query.filter_by(status="working").all()
+        min_required_reward = (
+            median_time_spent_in_hours(participants) * prolific_min_wage_per_hour
+        )
+        reward = participant.base_pay + participant.bonus
+
+        if reward < min_required_reward:
+            message = (
+                f"Participant with submission ID {participant.assignment_id} does not satisfy the requirements "
+                f"to be screened-out! Reward: {reward}, Minimum required reward: {min_required_reward}"
+            )
+            logger.warning(message)
+            return False
+
+        return True
+
```

Comment:
Once we're happy with the code you will need to test it with real participants, I think